cmd/util/cmd/common/identifer_list_arg.go

```diff
@@ -0,0 +1,33 @@
+package common
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// IdentifierListValue defines a custom command line spf13 pflag to accept a list of flow identifiers as a comma separated list of
+// hex string
+// e.g. -nodeid=b4a4dbdcd443d0ed1938bd3adec17aea9be65d8f0d57871bdd71ae8bc63d7fd1,fb386a6ad47f5c69022a335c0d78a94a77eee04bb5fe781bf3a6c092d72390ca
+type IdentifierListValue flow.IdentifierList
```

Comment:
The custom command line argument parser to parse the list of node ids into a flow.IdentifierList